layouts/partials/embeds/backup-locations.md

```diff
@@ -24,6 +24,7 @@ Before you specify to backup to an SFTP server, make sure that:
         to the SFTP server.
     - **Use a custom key** - Generate a TLS key pair for the SFTP server, copy the private key to
         the **SSH Private Key** box, and copy the public key to the SFTP server.
+    - The key shoud be added to ~/.ssh/authorized_keys of the user used for backup on SFTP server.
```

Comment:
@ushachar Should we tell customers where to put the key?

Reply:
No - it depends on which SFTP server they are running, and not something we can control / know.